README.md

```diff
@@ -0,0 +1,203 @@
+# Get started building your personal website
```

Comment:
Blitz this file please.

Reply:
I suppose I could rework the readme to provide more relevant sections. I was thinking instructions on how to create a blog entry or modify content would still be useful, or how to run this repo on your own machine?

Reply:
Blitzed.